ROMFS/tap_common/init.d/rcS

```diff
@@ -313,9 +313,9 @@ then
 fi
 
 #
-# Rover setup
+# UGV/Rover setup
 #
-if [ $VEHICLE_TYPE == rover ]
+if [ $VEHICLE_TYPE == rover -o $VEHICLE_TYPE == ugv]
```

Comment:
Do we need both VEHICLE_TYPEs?

Reply:
I'm glad you think that as well. I'll pull out the check for rover!

Reply:
As a note, when I implemented that logic update I searched the repo to see if anywhere in the code base `VEHICLE_TYPE` is set to `rover`, and there are no occurrences, so this was likely dead code previously anyway.